octreelib/ransac/util.py

```diff
@@ -0,0 +1,137 @@
+import math
+import numba as nb
+
+from numba import cuda
+from numba.cuda.random import xoroshiro128p_uniform_float32
+
+from octreelib.ransac.initial_points_config import N_INITIAL_POINTS
+
+
+@cuda.jit(
+    device=True,
+    inline=True,
+)
+def measure_distance(plane, point):
```

Comment:
Do we really need to store this util functions in separate file? They could be private scope in `CudaRansac`, wdyt?

Reply:
they can be in the file `cuda_ransac.py`, however they cannot be stored in `CudaRasnac` class because  `ransac_kernel` would not have access to these functions (cuda functions would not have access to `self` and `CudaRansac` class)

Reply:
added comments in the code about this issue